src/app/shared/input-suggestions/input-suggestions.component.ts

```diff
@@ -186,7 +186,7 @@ export class InputSuggestionsComponent implements ControlValueAccessor, OnChange
    * @param {KeyboardEvent} event The keyboard event
    */
   onKeydown(event: KeyboardEvent) {
-    if (event.key !== 'Enter') {
+    if (event.key !== 'Enter' && event.key !== 'Tab') {
```

Comment:
The comment above should be updated to include the Tab button in the parentheses.